app/policies/application_policy.rb

```diff
@@ -0,0 +1,49 @@
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def index?
+    false
+  end
+
+  def show?
+    false
+  end
+
+  def create?
+    false
+  end
+
+  def new?
```

Comment:
Should we define this despite it won't be used in APIs ?

Reply:
I would say yes, also this is an autogenerated file that is going to be inherited from other policies.
And time to time this base is used to implement webs too